package.json

```diff
@@ -8,12 +8,17 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "build": "webpack --config ./webpack.config.js --watch"
+    "build": "webpack --config ./webpack.config.js --watch",
+    "server": "nodemon server/server.js",
+    "seed": "node db/seed.js",
+    "db": "node db/db.js"
```

Comment:
@harrisonywu @krise3 I am uncertain about what the purpose of the `db` script is.